.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "talawa api dev environment",
+  "dockerComposeFile": "docker-compose.yaml",
```

Comment:
shouldn't you utilize the already existing `docker-compose.dev.yaml` file?

Reply:
When I was using the compose file outside the  .decontainer.json file it was having some issues related to the project directory, that's why I used the file inside it. If I move the already existing file to that folder, developers can have issues, if they use docker for db.